middle_end/flambda2/parser/fexpr_to_flambda.ml

```diff
@@ -878,10 +878,15 @@ let rec expr env (e : Fexpr.expr) : Flambda.Expr.t =
         let cost_metrics =
           Cost_metrics.from_size (Code_size.of_int code_size)
         in
+        let param_modes =
+          List.map
+            (fun _ -> Alloc_mode.For_types.heap)
```

Comment:
Why are these all heap?

Reply:
There are a lot of features of flambda that are not reflected in fexpr, and this is one of them. However, it might be better to reflect the modes here as well, but then it would require changing the syntax.